src/adapter/src/flags.rs

```diff
@@ -88,6 +88,11 @@ pub fn storage_config(config: &SystemVars) -> StorageParameters {
             tcp_user_timeout: Some(config.pg_source_tcp_user_timeout()),
         },
         pg_source_snapshot_statement_timeout: config.pg_source_snapshot_statement_timeout(),
+        mysql_source_timeouts: mz_mysql_util::TimeoutConfig {
+            tcp_keepalive: Some(config.mysql_source_tcp_keepalive()),
+        },
+        mysql_source_snapshot_max_execution_time: config.mysql_source_snapshot_max_execution_time(),
+        mysql_source_snapshot_lock_wait_timeout: config.mysql_source_snapshot_lock_wait_timeout(),
```

Comment:
Ah I was matching the semantics of the pg configurations which keeps the `pg_source_snapshot_statement_timeout` separate from the tcp_timeouts since the former is directly read in the source implementation whereas the latter is used by the `mz_postgres_util::Config`

Reply:
simplified to use a single struct for all the params